pkg/cmd/cli/cmd/logs.go

```diff
@@ -189,5 +218,32 @@ func (o OpenShiftLogsOptions) RunLog() error {
 		// Use our own options object.
 		o.KubeLogOptions.Options = o.Options
 	}
-	return o.KubeLogOptions.RunLogs()
+	if !o.JenkinsLogs {
+		return o.KubeLogOptions.RunLogs()
+	}
+
+	bc, ok := o.KubeLogOptions.Object.(*buildapi.BuildConfig)
+	var urlString string
+	if !ok {
+		build, ok := o.KubeLogOptions.Object.(*buildapi.Build)
+		if ok {
+			urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+		}
+	} else {
+		buildName := buildutil.BuildNameForConfigVersion(bc.ObjectMeta.Name, int(bc.Status.LastVersion))
+		build, _ := o.Client.Builds(o.Namespace).Get(buildName, metav1.GetOptions{})
+		if build != nil {
+			urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+		}
+	}
+	if len(urlString) > 0 {
+		o.KubeLogOptions.Out.Write([]byte("Because this is a Pipeline Strategy build, the logs are accessed via the Jenkins server:"))
+		o.KubeLogOptions.Out.Write([]byte("\n"))
+		o.KubeLogOptions.Out.Write([]byte(urlString))
+		o.KubeLogOptions.Out.Write([]byte("\n"))
+	} else {
+		return errors.New("Could not find the log URL for this Jenkins Pipeline Strategy build")
```

Comment:
another issue with this error is it's misleading.  You could also get here if we didn't find any build associated w/ the buildconfig (if the user passed a BC and not a Build).  That should be handled as a separate condition/error.  

(It seems a bit odd that we must have this logic (looking up the build for a bc) in two places in the log flow now, once for "normal" builds and once for pipeline builds.  Are we making the distinction too early in the code?  Could/should we be doing that down in RunLogs() instead?)